restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/RCDevice.java

```diff
@@ -649,22 +631,6 @@ public void release()
 
```

Comment:
Shouldn't you still close() signaling client here before the device state is set to offline?

Reply:
Sure, thanks.